spec/factories/merchant.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :merchant do
+    name { Faker::Kpop.boy_bands }
```

Comment:
Here's a list of Faker to use (something for fun)

https://github.com/faker-ruby/faker